src/SelectPanel/SelectPanel.tsx

```diff
@@ -0,0 +1,107 @@
+import React, {useCallback, useMemo} from 'react'
+
+import {FilteredActionList, FilteredActionListProps} from '../FilteredActionList'
+import {OverlayProps} from '../Overlay'
+import {ItemInput} from '../ActionList/List'
+import {FocusZoneSettings} from '../behaviors/focusZone'
+import {DropdownButton, DropdownButtonProps} from '../DropdownMenu'
+import {ItemProps} from '../ActionList'
+import {AnchoredOverlay} from '../AnchoredOverlay'
+import Flex from '../Flex'
+
+export interface SelectPanelProps extends Omit<FilteredActionListProps, 'onChange'> {
+  open: boolean
+  onOpen: (gesture: 'anchor-click' | 'anchor-key-press') => unknown
+  onClose: (gesture: 'click-outside' | 'escape') => unknown
+  renderAnchor?: <T extends React.HTMLAttributes<HTMLElement>>(props: T) => JSX.Element
+  placeholder?: string
+  selectedItems: ItemInput[]
+  selectionVariant?: 'single' | 'multiple'
+  onChange?: (selectedItems: ItemInput[]) => unknown
+  onFilterChange: (e: React.ChangeEvent<HTMLInputElement>) => unknown
+  overlayProps?: Partial<OverlayProps>
+}
+
+const focusZoneSettings: Partial<FocusZoneSettings> = {
+  focusOutBehavior: 'wrap',
+  focusableElementFilter: element => {
+    return !(element instanceof HTMLInputElement) || element.type !== 'checkbox'
+  }
+}
+
+export function SelectPanel({
+  open,
+  onOpen,
+  onClose,
+  renderAnchor = <T extends DropdownButtonProps>(props: T) => <DropdownButton {...props} />,
```

Comment:
```suggestion
  renderAnchor = DropdownButton,
```
For my own learnings, is this line equivalent?

Reply:
I've simplified it to `props => <DropdownButton {...props} />`, but I think that's the smallest we can go.  Because of the forwarded ref, simple `DropdownButton` by itself has an `| null` in the return type.